src/libraries/System.Runtime.InteropServices.JavaScript/tests/JsImportGenerator.UnitTest/JSImportGenerator.Unit.Tests.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
naming nit: Can we unify folder name and csproj name?

Reply:
keeping it consistent with LibraryImport